src/OceanSeaIceModels/PrescribedAtmospheres.jl

```diff
@@ -1,7 +1,7 @@
 module PrescribedAtmospheres
 
 using Oceananigans.Grids: grid_name
-using Oceananigans.Utils: prettysummary
+using Oceananigans.Utils: prettysummary, Time
```

Comment:
this is the only change here so it may not be used

Reply:
There is a bug in `PrescribedAtmospheres` where that `Time` is missing.